src/ai/backend/manager/config.py

```diff
@@ -307,11 +308,58 @@
             t.Key("log-scheduler-ticks", default=False): t.ToBool,
             t.Key("periodic-sync-stats", default=False): t.ToBool,
         }).allow_extra("*"),
+        t.Key("raft", default=None): t.Null
+        | t.Dict({
+            # Storage configurations
+            t.Key("log-dir"): t.String,
+            # Raft core configurations
+            # TODO: Decide proper default values for these configs.
+            t.Key("heartbeat-tick", default=None): t.Int | t.Null,
+            t.Key("election-tick", default=None): t.Int | t.Null,
+            t.Key("min-election-tick", default=None): t.Int | t.Null,
+            t.Key("max-election-tick", default=None): t.Int | t.Null,
+            t.Key("max-committed-size-per-ready", default=None): t.Int | t.Null,
+            t.Key("max-size-per-msg", default=None): t.Int | t.Null,
+            t.Key("max-inflight-msgs", default=None): t.Int | t.Null,
+            t.Key("check-quorum", default=None): t.ToBool | t.Null,
+            t.Key("batch-append", default=None): t.ToBool | t.Null,
+            t.Key("max-uncommitted-size", default=None): t.Int | t.Null,
+            t.Key("skip-bcast-commit", default=None): t.ToBool | t.Null,
+            t.Key("pre-vote", default=None): t.ToBool | t.Null,
+            t.Key("priority", default=None): t.Int | t.Null,
+        }).allow_extra("*"),
     })
     .merge(config.etcd_config_iv)
     .allow_extra("*")
 )
 
+manager_raft_cluster_config_iv = t.Dict({
+    t.Key("restore-wal-from", default=None): t.Int | t.Null,
+    t.Key("restore-wal-snapshot-from", default=None): t.Int | t.Null,
+    t.Key("raft-debug-webserver-enabled", default=False): t.ToBool,
+    t.Key("peers"): t.Dict({
+        t.Key("myself"): t.List(
+            t.Dict({
+                t.Key("node-id"): t.Int,
+                t.Key("host"): t.String,
+                t.Key("port"): t.Int,
+                t.Key("role", default=RaftNodeInitialRole.VOTER): tx.Enum(RaftNodeInitialRole),
+            })
+        ),
+        t.Key("other", default=[]): (
+            t.List(
+                t.Dict({
+                    t.Key("node-id"): t.Int,
+                    t.Key("host"): t.String,
+                    t.Key("port"): t.Int,
+                    t.Key("role", default=RaftNodeInitialRole.VOTER): tx.Enum(RaftNodeInitialRole),
+                })
+            )
+            | t.Null
+        ),
+    }),
+}).allow_extra("*")
+
```

Comment:
Thanks for your correction. I think there is a accident while merging with main branch.
I will revert this.

Reply:
The contents of `raft-config-cluster.toml` may be updated automatically in the future based on further implementations (although it has not yet been implemented).

This is intended to update only the information about the cluster members that need updating, rather than overwriting the entire `manager.toml`.